.cbmc-batch/source/make_common_data_structures.c

```diff
@@ -75,20 +75,10 @@ bool aws_priority_queue_is_bounded(
     return container_is_bounded && backpointers_is_bounded;
 }
 
-bool ensure_priority_queue_has_allocated_members(struct aws_priority_queue *const queue) {
+void ensure_priority_queue_has_allocated_members(struct aws_priority_queue *const queue) {
```

Comment:
Thanks for this change, this is a lot more sensible. So I suppose the idea is that we should never be using `ensure_*` functions inside a `__CPROVER_assume()`?

Do all the other `ensure_*` functions have void return type too? If not, it might be worth opening a PR to change all of them and fix their callers at the same time.

Reply:
You are right. Every other `ensure_*` is used in this manner, so there is no change to be made.